compute/disk-images/disk_image_test.go

```diff
@@ -136,4 +141,30 @@ func TestComputeDiskImageSnippets(t *testing.T) {
 	if err != nil {
 		t.Errorf("deleteDisk got err: %v", err)
 	}
+
+	t.Run("Test creation and deletion from image", func(t *testing.T) {
+		buf.Reset()
+		want := "created"
+		image, err2 := getDiskImageFromFamily(buf, sourceProjectId, sourceImageName)
+		if err2 != nil {
+			t.Fatalf("getDiskImageFromFamily got err: %v", err2)
+		}
+
+		if err := createImageFromImage(buf, tc.ProjectID, sourceProjectId, *image.Name, imageName, []string{}, guestOsFeatures); err != nil {
+			t.Fatalf("createImageFromImage got err: %v", err)
```

Comment:
```suggestion
			t.Errorf("createImageFromImage got err: %v", err)
```
Error, otherwise delete does not run and resources are orphaned.